CONTRIBUTING.md

```diff
@@ -381,6 +381,68 @@ It offers a number of functionalities that improve development:
   relative import from the file system. As a consequence, the mechanism for
   overriding import paths for tests and Storybook's pages will not work.
 
+## Unit tests
+
+The project has unit tests for both C++ and QML code.
+
+### C++ unit tests
+
+C++ unit tests are localized under `ui/StatusQ/tests`. They can be executed
+easily from QtCreator. Just open the `StatusQ` project in `QtCreator`
+(`status-desktop/ui/StatusQ/CMakeLists.txt`) and select the appropriate target (each test is a separate target).
+
+### QML unit tests
+
+QML tests are localized under `storybook/qmlTests/tests` and `ui/StatusQ/tests`. The
+first location is recommended for all new tests because it provides integration with
+the `Storybook`.
+
+Unit tests operate without backend. Backend layer is cut-off using stubs mechanism in the same
+way as for pages in `Storybook`.
+
+QML tests can be executed by running `QmlTests` target of `Storybook` project
```

Comment:
just realised there's no option of running a specific test using the make target. We're missing the `ARGS` option. I'm usually using something like `storybook/build/bin/QmlTests -input /Users/alexjbanca/Repos/status-desktop/storybook/qmlTests/tests/tst_ConnectDAppModal.qml` but won't work without building the tests before. Maybe something to be easily added for contributors that are not using QtCreator.

Reply:
It's not enough to add `${ARGS}` because the test is run via `ctest` and it seem that there is no simple way to pass sth to the actual test...